build

```diff
@@ -14,13 +14,15 @@ export GOPATH=${PWD}/gopath
 eval $(go env)
 
 echo "Building network plugins"
-for d in networking/net/* networking/ipam/*; do
-	if [ -d $d ]; then
-		plugin=$(basename $d)
-		echo "  " $plugin
-		go install ${REPO_PATH}/$d
-	fi
-done
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
```

Comment:
Can you just move this into the existing linux section (e.g. line 30 on). Also the echo should move within the conditional.